resources/css/app.css

```
@@ -88,6 +93,10 @@
 @theme inline {
     --font-sans: 'Manrope Variable', sans-serif;
     --font-heading: 'Montserrat Variable', sans-serif;
+    --font-serif: 'Instrument Serif', ui-serif, Georgia, 'Times New Roman', serif;
```

Stylelint (17.7.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)